pkg/accounting/tally/tally_test.go

```diff
@@ -69,7 +69,7 @@ func sendGeneratedAgreements(ctx context.Context, t *testing.T, db satellite.DB,
 
 	agreements := make([]*psdb.Agreement, len(actions))
 	for i, action := range actions {
-		pba, err := testbwagreement.GeneratePayerBandwidthAllocation(action, satID, upID, time.Hour)
+		pba, err := testbwagreement.GenerateFundsOrder(action, satID, upID, time.Hour)
```

Comment:
`OrderLimit` makes more sense, because we are not actually sending funds, but rather setting a transfer limit. Funds / payment calculation is entirely separate.